app/component/project_components/project_card_component.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module ProjectComponents
+  class ProjectCardComponent < ViewComponent::Base
```

Comment:
Are we using the project card everywhere in the codebase? If no we might have to specify this in the name itself. Like `ProjectComponents::SearchResult::ProjectCard`

Reply:
not now but later once the UI of card is revamped we can use this in other places as well such as Explore Page on which Aaditya will be working, or should i just move this to Search Components ? 
`SearchComponents::ProjectCard`